frontends/web/src/routes/account/send/send.module.css

```diff
@@ -109,4 +109,11 @@
     font-size: var(--size-small);
     font-weight: 400;
     margin-top: 0;
+}
+
+.spinnerAnimationContainer {
+    align-items: center;
+    display: flex;
+    height: 300px;
```

Comment:
Matching the QR dialog's height when loading is finished (has video) to avoid UI shift/jumps.